content/id/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -0,0 +1,374 @@
+* `periodSeconds`: Seberapa sering (dalam detik) _probe_ dijalankan. Nilai bawaannya adalah 10 detik.
+Nilai minimalnya adalah 0.
+* `timeoutSeconds`: Durasi dalam detik setelah _probe_ mengalami _timeout_. Nilai bawaannya adalah 1 detik.
+Nilai minimalnya adalah 0.
+* `successThreshold`: Jumlah minimal sukses yang berurutan untuk _probe_ dianggap berhasil
+setelah mengalami kegagalan. Nilai bawaannya adalah 1. Nilanya harus 1 untuk _liveness_.
+Nilai minimalnya adalah 1.
+* `failureThreshold`: Ketika sebuah Pod dimulai dan _probe_ mengalami kegagalan, Kubernetes
+akan mencoba beberapa kali sesuai nilai `failureThreshold` sebelum menyerah. Menyerah karena
+kasus _probe liveness_ akan membuat Container diulang kembali. Untuk _probe readiness_, menyerah
+akaan menandai Pod menjadi "tidak siap" (Unready). Nilai bawaannya adalah 3. Nilai minimalnya adalah 1.
+
+[_Probe_ HTTP](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#httpgetaction-v1-core)
+memiliki _field-field_ tambahan yang bisa diatur melalui `httpGet`:
+
+* `host`: Nama dari host yang akan terhubung, nilai bawaannya adalah IP dari Pod. Kamu mungkin
+juga ingin mengatur "Host" pada httpHeaders.
+* `scheme`: Skema yang digunakan untuk terhubung pada host (HTTP atau HTTPS). Nilai bawaannya adalah HTTP.
+* `path`: _Path_ untuk mengakses server HTTP.
+* `httpHeaders`: _Header_ khusus yang diatur melalui permintaan. HTTP memperbolehkan _header_ yang berulang.
+* `port`: Nama atau angka dari porta untuk mengakses Container. Angkanya harus ada di antara 1 sampai 65535.
+
+Untuk sebuah _probe_ HTTP, kubelet mengirimkan permintaan HTTP untuk _path_ yang ditentukan
+dan porta untuk mengerjakan pemeriksaan. kubelet mengirimkan _probe_ untuk alamat IP Pod,
+kecuali saat alamat digantikan oleh _field_ opsional pada `httpGet`. Jika _field_ `scheme`
+diatur menjadi `HTTPS`, maka kubelet mengirimkan permintaan HTTPS dan melewati langkah verifikasi
+sertifikat. Pada skenario kebanyakan, kamu tidak menginginkan _field_ `host`.
+Berikut satu skenario yang memerlukan `host`. Misalkan Container mendengarkan permintaan
+melalui 127.0.0.1 dan _field_ `hostNetwork` pada Pod bernilai true. Kemudian `host`, melalui
+`httpGet`, harus diatur menjadi 127.0.0.1. Jika Pod kamu bergantung pada host virtual, dimana
+untuk kasus-kasus umum, kamu tidak perlu menggunakan `host`, tetapi perlu mengaatur _header_
+`Host` pada `httpHeaders`.
+
+Untuk _probe_ TCP, kubelet membuat koneksi _probe_ pada Node, tidak pada Pod, yang berarti bahwa
+kamu tidak menggunakan nama Service di dalam parameter `host` karena kubelet tidak bisa
+me-_resolve_-nya.
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* Pelajari lebih lanjut tentang
+[Probe Container](/id/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
+
+Kamu juga dapat membaca rujukan API untuk:
+
+* [Pod](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core)
```

Comment:
Do we need to link to Indonesian documentation? In order to be consistent with line 366.

Reply:
we don't have id pages for the API pages yet